tests/components/home_connect/test_light.py

```diff
@@ -0,0 +1,195 @@
+"""Tests for home_connect light entities."""
```

Comment:
I recommend only including tests for one platform and do the remaining platforms later one platform per PR. We don't need to include it all in one PR. It will just delay review and merge.

Reply:
I left only the `sensor` platform tests.